test

```diff
@@ -11,6 +11,9 @@ LINTABLE=$( go list -tags=golint -f '
 
 go test -v -i -race github.com/coreos/go-oidc/...
 go test -v -race github.com/coreos/go-oidc/...
-golint $LINTABLE
+if ! golint $LINTABLE &> /dev/null
+then
+  echo "golint not installed, skipping linting"
```

Comment:
golint is long deprecated, why are we adding it here?

https://github.com/golang/go/issues/38968

Reply:
I was not adding it, I was just trying to do a minimal change here to get it working.  I felt like making it optional was a good first step.

Reply:
Gotcha